src/main/java/seedu/address/model/task/RecurType.java

```diff
@@ -4,16 +4,28 @@
     DAILY {
         @Override
         public long getInterval() {
-            // return 1000l*60*60*24;
-            return 15000l; // for testing
+            return 1000l * 60 * 60 * 24;
+            // return 60000l; // for testing
+        }
+
+        @Override
+        public long getDayInterval() {
+            return 1;
         }
     },
     WEEKLY {
         @Override
         public long getInterval() {
             return 1000l * 60 * 60 * 24 * 7;
         }
+
+        @Override
+        public long getDayInterval() {
+            return 7;
```

Comment:
Same as the earlier comment about "magic numbers". For your consideration.

Reply:
Fixed